queries/corn/highlights.scm

```diff
@@ -9,7 +11,12 @@
   "]"
 ] @punctuation.bracket
 
-"." @punctuation.delimiter
+[
+  "."
+  "="
```

Comment:
is `=` not an operator?

Reply:
in this language i felt like it was more of a delimiter, like `:` in js objects, but i think treating it as an operator makes sense too. also i only found one other place where `=` is treated as `@punctuation.delimiter`